src/exporter/DaemonMetricCollector.cc

```diff
@@ -82,35 +82,41 @@ std::string boost_string_to_std(boost::json::string js) {
 
 std::string quote(std::string value) { return "\"" + value + "\""; }
 
-void DaemonMetricCollector::dump_asok_metrics() {
+void DaemonMetricCollector::dump_asok_metrics(bool sort_metrics, int64_t counter_prio,
+                                              bool sockClientsPing, std::string dump_response,
+                                              std::string schema_response,
+                                              bool config_show_response) {
```

Comment:
```suggestion
void DaemonMetricCollector::dump_asok_metrics(bool sort_metrics, int64_t counter_prio,
                                              bool sockClientsPing, std::string &dump_response,
                                              std::string &schema_response,
                                              bool config_show_response) {
```
Use references instead, copying strings is slow.